### ISSUE TITLE
500 Server Error due to cURL

### ISSUE THREAD

**Issue by MehulBawadia** [MehulBawadia/kshows#39]
The application is hosted on a shared server of Honstinger.

The issue is sometimes, the application gives a `500 Server Error` when trying to visit a page. 
And sometimes it just works fine without any errors.

This, however. does not happen on my local development machine. It is happening only after deploying it live.

The exact error is:

```log
production.ERROR: cURL error 35: Recv failure: Connection reset by peer (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://api.themoviedb.org/3/discover/movie?api_key=<myApiKey>&page=1&sort_by=primary_release_date.desc&vote_average.gte=0&vote_average.lte=10&with_original_language=ko&watch_region=KR&release_date.lte=2024-04-17&release_date.gte=1970-01-01&vote_count.gte=3 at /path/to/project-root/vendor/laravel/framework/src/Illuminate/Http/Client/PendingRequest.php:926)
```

I have also consulted Hostinger's Customer Support, but they could not guide me to a solution. Hence, I am stuck.

**Comment by MehulBawadia** [comment]
This issue won't be fixed. Added the label `wontfix`. Hence, closing it.
